test/phone_form_field_test.dart

```diff
@@ -697,6 +699,22 @@ void main() {
         await tester.pumpAndSettle();
         expect(find.text(national), findsNothing);
       });
+
+      testWidgets(
+          'Should cut off the TextField value when limitLength is true and a long nsn for the selected country is provided',
```

Comment:
nit:

"Should cut off the TextField value when limitLength is true and a long nsn for the selected country is provided"

To

"Should cut off the TextField value when limitLength is true and an nsn longer than the limit for the selected country is provided"